crates/swift-bridge-macro/tests/ui/args-into-argument-not-found.rs

```diff
@@ -15,11 +15,13 @@ mod ffi {
     }
 }
 
+#[allow(unused)]
 fn some_function(arg: u8) {}
 
 struct SomeType;
 
 impl SomeType {
+    #[allow(unused)]
     fn some_method(&self, foo: u8) {}
```

Comment:
Added `#[allow(unused)]` to fix the following warnings.
```
error: Argument "arg_typo" was not found in "fn some_function(..)"
 --> tests/ui/args-into-argument-not-found.rs:7:42
  |
7 |         #[swift_bridge(args_into = (arg, arg_typo))]
  |                                          ^^^^^^^^

error: Argument "bar" was not found in "fn some_method(..)"
  --> tests/ui/args-into-argument-not-found.rs:13:42
   |
13 |         #[swift_bridge(args_into = (foo, bar))]
   |                                          ^^^

warning: unused variable: `arg`
  --> tests/ui/args-into-argument-not-found.rs:19:18
   |
19 | fn some_function(arg: u8) {}
   |                  ^^^ help: if this is intentional, prefix it with an underscore: `_arg`
   |
   = note: `#[warn(unused_variables)]` on by default

warning: unused variable: `foo`
  --> tests/ui/args-into-argument-not-found.rs:25:27
   |
25 |     fn some_method(&self, foo: u8) {}
   |                           ^^^ help: if this is intentional, prefix it with an underscore: `_foo`
```

Reply:
Instead of `#[allow(unused)]` let's do `_arg` and `_foo`. Less noisy.